modules/unsupported/css/src/main/java/org/geotools/styling/css/CssTranslator.java

```diff
@@ -219,6 +215,27 @@
         }
     };
 
+    @SuppressWarnings("serial")
+    static final Map<String, String> CONTRASTENHANCMENT_VENDOR_OPTIONS = new HashMap<String, String>() {
+        {
+            put("-gt-raster-contrast-enhancement-algorithm", "algorithm");
+
+            put("-gt-raster-contrast-enhancement-minvalue", "minValue");
+            put("-gt-raster-contrast-enhancement-maxvalue", "maxValue");
+
+            put("-gt-raster-contrast-enhancement-normalizationFactor", "normalizationFactor");
+            put("-gt-raster-contrast-enhancement-correctionFactor", "correctionFactor");
+            //short forms for lazy people 
+            put("-gt-rce-algorithm", "algorithm");
+
+            put("-gt-rce-minvalue", "minValue");
+            put("-gt-rce-maxvalue", "maxValue");
```

Comment:
Not against it, just not sure I have seen a short form used before? "-gt-normalization-min" would be my pick :)